.github/workflows/ci.yml

```diff
@@ -78,27 +78,23 @@ jobs:
         run: make test
 
       - name: Upload cnd-${{ matrix.os }} archive that contains the cnd binary
+        if: matrix.os == 'ubuntu-latest'
         uses: actions/upload-artifact@v1
         with:
-          name: cnd-${{ matrix.os }}
+          name: cnd
           path: target/debug/cnd
 
   e2e_test:
-    strategy:
-      # Do not cancel ubuntu even if macos fails
-      fail-fast: false
-      matrix:
-        os: [macos-latest, ubuntu-latest]
```

Comment:
Could have also just removed `macos-latest` here, no?

Reply:
@bonomat please note I privileged this method to keep the nice `(ubuntu-latest)` after the GitHub check name to not have to modify all bors/mergify/github branch protection configurations.

Reply:
@thomaseizinger 
> Could have also just removed macos-latest here, no?

probably :D